Sources/JSONWebToken/Data+hash.swift

```diff
@@ -40,7 +40,7 @@ extension Data {
         }
     }
     func hmac(_ hashFunction: SignatureAlgorithm.HashFunction, secret: Data) -> Data {
-        if #available(iOSApplicationExtension 13.0, *) {
```

Comment:
this is a fairly significant change as it means this can no longer be used in an app extension (such as the share extension). Is that really necessary?